<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiSigPage</class>
 <widget class="QWidget" name="MultiSigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>529</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <widget class="QLabel" name="contactLabel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>34</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
font: 14pt &quot;微软雅黑&quot;;
color: rgb(40, 40, 40);</string>
   </property>
   <property name="text">
    <string>多重签名</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="toSignTextEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入待签名的交易</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>交易信息</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="transactionInfoTextBrowser">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>选择签名账户</string>
   </property>
  </widget>
  <widget class="QComboBox" name="accountComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>账户地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="signBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>签名</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="afterSignTextBrowser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>签名后交易</string>
   </property>
  </widget>
  <widget class="QToolButton" name="broadcastBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>广播交易至区块链网络</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
